src/Client/Connection.cpp

```diff
@@ -551,6 +551,15 @@ void Connection::sendIgnoredPartUUIDs(const std::vector<UUID> & uuids)
     out->next();
 }
 
+
+void Connection::sendReadTaskResponse(const std::optional<String> & response)
+{
+    writeVarUInt(Protocol::Client::ReadTaskResponse, *out);
+    writeVarUInt(DBMS_CLUSTER_PROCESSING_PROTOCOL_VERSION, *out);
+    writeStringBinary(response.has_value() ? String(*response) : "", *out);
```

Comment:
If our protocol does not distinguish `std::null_opt` and empty string, maybe we can remove `std::optional` at all?

Reply:
std::optional is very convenient, because you can write something like 'while (auto task = iterator.next()) {}'

Reply:
But I think we could remove this.

Reply:
ok